### ISSUE TITLE
Too many labels in grid column

### ISSUE THREAD

**Issue by anxgstadler** [z-song/laravel-admin#5723]
- Laravel Version: 9
- PHP Version: 8
- Laravel-admin: 1.8.19

### Description:
When there are too many labels for a Grid column, the column grows and grows in width without breaking to a second or even third line. This is caused by using non-breaking spaces as separator in ```Encore\Admin\Grid\Displayers\Label```.

We are using laravel admin in a fairly complex project with many permissions, and the "Roles" page has become quite unusable by now because every line is longer than one or even two screens ;-)

Proposed changes:
1) Use regular space as separator
2) Since this would be a BC issue for some users, we could introduce a second optional parameter ```bool $allowLineBreaksBetweenLabels = false``` that could be set to ```true``` in cases where you want to allow line breaks between labels (like on the "Roles" page)

I will happily provide a small pull request if the feature is met with approval.

**Comment by alexoleynik0** [comment]
I agree that this is a bit annoying, but it can be "fixed" by adding `max-width and word-break` to the column.
```php
$grid->column('tags', __('Tags'))
    ->pluck('name')
    ->label('default')
    ->style('max-width: 220px; word-break: break-word;')
;
```

**Comment by anxgstadler** [comment]
This in indeed a nice idea.
The thing is: I'm talking of the "Roles" page within laravel admin itself. So I need to do it via pull request anyway, either changing only this one page, or addressing the root of the cause.

**Comment by alexoleynik0** [comment]
Oh, I see. Don't know if there's a reason to use `&nbsp;` between some html elements.. Maybe there's one, who knows.
